src/Grpc.Core.Api/Grpc.Core.Api.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>gRPC C# Surface API</Description>
+    <PackageTags>gRPC RPC HTTP/2</PackageTags>
+
+    <IsGrpcPublishedPackage>true</IsGrpcPublishedPackage>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <TargetFrameworks>net45;netstandard1.5;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
net45 isn't supported anymore. See https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-framework

Should change net45 to net462 for various Grpc.Core projects.

Reply:
ok, that seems doable. But there should also be a some follow-up cleanup after switching to net462 (`#if` defs and also there is some compatibility code that is no longer needed on net462)